src/test/java/uk/gov/companieshouse/controller/DissolutionControllerTest.java

```diff
@@ -39,13 +45,26 @@
     private static final String AUTHORISED_USER_HEADER = "ERIC-Authorised-User";
 
     private static final String COMPANY_NUMBER = "12345678";
+    private static final String COMPANY_NAME = "ComComp";
     private static final String USER_ID = "1234";
     private static final String EMAIL = "user@mail.com";
     private static final String IP_ADDRESS = "127.0.0.1";
 
     @MockBean
     private DissolutionService service;
 
+    @MockBean
+    private CompanyProfileService companyProfileService;
+
+    @MockBean
+    private CompanyOfficerService companyOfficerService;
```

Comment:
These are not injected into the controller and should therefore not appear in the tests